# a.
# print('Begin')
# x = int(input())
# print(x)
# print('End')
#i. User enters 22
#ii. User enters ZZ 'EXCEPTION'

# b.
# print('Begin')
# try:
#     x = int(input())
#     print(x)
# except ValueError:
#     print('Wrong!')
#     print('End')
# i. User enters 22
# ii. User enters ZZ 'EXCEPTION'

# c.
# print('Begin')
# try:
#     x = int(input())
#     print(x)
# except IndexError:
#     print('Wrong!')
#     print('End')
# i. User enters 22
# ii. User enters ZZ 'EXCEPTION'

# d.
# print('Begin')
# try:
#     x = int(input())
#     print(x)
# except Exception:
#     print('Wrong!')
#     print('End')
# i. User enters 22
# ii. User enters ZZ

# e.
# print('Begin')
# try:
#     x = int(input())
#     print(x)
# except ValueError:
#     print('Wrong!')
# else:
#     print('Wow')
#     print('End')
# i. User enters 22
# ii. User enters ZZ

# f.
# print('Begin')
# try:
#     x = int(input())
#     print(x)
# except ValueError:
#     print('Wrong!')
# finally:
#     print('Done')
#     print('End')
# i. User enters 22
# ii. User enters ZZ

# g.
# print('Begin')
# try:
#     x = int(input())
#     print(x)
# except ValueError:
#     print('Wrong!')
# else:
#     print('Wow')
# finally:
#     print('Done')
#     print('End')
# i. User enters 22
# ii. User enters ZZ